mcdreforged/plugin/server_interface.py

```diff
@@ -31,6 +32,21 @@
 SerializableType = TypeVar('SerializableType')
 
 
+class ServerInfo:
+	version = None
+	ip = None
+	port = None
+
+	def set_ip(self, ip: str, port: int):
+		if not 0 <= port <= 65535:
```

Comment:
I doubt if the check here is necessary:
- If you want to check the legity of the port, why not check it inside the handler
- If you still want to have a check here, why don't you check the address